app/lib/backend_api_logic/routing_policy.rb

```diff
@@ -37,7 +37,7 @@ def to_a
       end
 
       class Rule
-        delegate :private_endpoint, :path, to: :@config
+        delegate :private_endpoint, :config_path, to: :@config
```

Comment:
Imo 'config_path' shouldn't be a method belonging to BackendApiConfig but belonging to this specific class, like this:
![image](https://user-images.githubusercontent.com/11318903/66513543-a5170f00-eadb-11e9-9519-79c41b52cc58.png)

And then the test:
![image](https://user-images.githubusercontent.com/11318903/66513576-b8c27580-eadb-11e9-95de-ff70f7ad7e5b.png)

'config_path' is not data itself but manipulation of it, as business logic, and needed only for this class, so i that's why i would put it here instead of in the model.